sumpy/expansion/diff_op.py

```diff
@@ -65,6 +65,13 @@ def __eq__(self, other):
     def __hash__(self):
         return hash((self.dim, self.eqs))
 
+    @property
+    def degree(self):
```

Comment:
I think "order" is the more conventional term. Would that work for you?

Reply:
Yes, works for me.